icepyx/core/query.py

```diff
@@ -916,11 +916,23 @@ def visualize_spatial_extent(
         >>> reg_a.visualize_spatial_extent
         [visual map output]
         """
+        gdf = geospatial.geodataframe(self.extent_type, self._spat_extent)
 
-        world = gpd.read_file(gpd.datasets.get_path("naturalearth_lowres"))
-        f, ax = plt.subplots(1, figsize=(12, 6))
-        world.plot(ax=ax, facecolor="lightgray", edgecolor="gray")
-        geospatial.geodataframe(self.extent_type, self._spat_extent).plot(
-            ax=ax, color="#FF8C00", alpha=0.7
-        )
-        plt.show()
+        try:
+            from shapely.geometry import Polygon
+            import geoviews as gv
+            gv.extension('bokeh')
```

Comment:
The `black` linter suggests adding a line between `import` statements and the line below.
```suggestion
            from shapely.geometry import Polygon
            import geoviews as gv

            gv.extension('bokeh')
```